### ISSUE TITLE
[Feature Request]: Allow administrators to make changes db filed name of Custom Filed

### ISSUE THREAD

**Issue by kenchan0130** [grokability/snipe-it#15423]
### Is your feature request related to a problem? Please describe.

Currently, the Custom Filed code is automatically generated from the field name.
The first problem is the code is `_snipeit__11` for non-ASCII languages.
As a use case, if an error occurs when importing from CSV, the code is displayed as an error, but there is a problem of not knowing which filed it is.

Also, when adding a value to a custom field from the API, even if the field name is the same, the code will change, so if someone deletes and adds a custom field with the same name, you will need the code,
code is required, which is a hassle.

### Describe the solution you'd like

Allow administrators to make changes code of Custom Filed.

### Describe alternatives you've considered

Tell admin members not to use English in field names and not to implement field additions and deletions

**Comment by snipe** [comment]
> you will need the code, code is required, which is a hassle.

I don't know what you mean by code here. We show you the field name in the custom fields section.

<img width="728" alt="Manage_Custom_Fields____Snipe-IT_Demo" src="https://github.com/user-attachments/assets/28655c02-435e-49e0-9d69-65e7fa5ecc05">

Also none of this is particularly useful without knowing what version you're running.

**Comment by kenchan0130** [comment]
> I don't know what you mean by code here. We show you the field name in the custom fields section.

I am sorry. I meant to indicate DB field name.

<img width="1280" alt="スクリーンショット 0006-08-31 0 54 45" src="https://github.com/user-attachments/assets/9eb23311-836b-40b4-9964-27a6e42a2402">

I am using `Version v7.0.11 - build 15044 (master)`.

**Comment by snipe** [comment]
I don't think this is something we'll be adding. It doesn't come up often enough and the naming scheme we use it important to the way the system works.
